p2p/swarm_test.go

```diff
@@ -1370,8 +1370,8 @@ func (u *UpnpGatewayMock) Clear(port uint16) error {
 	return nil
 }
 
-func createDiscoverUpnpFunc(funcErr error, port uint16, gatewayForwardErr error) func() (igd nat_traversal.UpnpGateway, err error) {
-	return func() (igd nat_traversal.UpnpGateway, err error) {
+func createDiscoverUpnpFunc(funcErr error, port uint16, gatewayForwardErr error) func() (igd nattraversal.UpnpGateway, err error) {
+	return func() (igd nattraversal.UpnpGateway, err error) {
```

Comment:
some places we use `Upnp` and some places we use `UPnP` (`discoverUPnPGateway`). lets try to be consistent ? I know using a lot of capital letters might not look the best. but usually acronyms should be written with capital letters..

Reply:
Good point. Fixed this.